### DISCUSSION TITLE
Server direct to Chromecast streaming

### DISCUSSION THREAD

**Discussion by grolschie** [ciderapp/Cider#979] [Feature Request]
Hi. I'm not sure if it's possible with MusicKit, but when Chromecasting from Cider, is it possible to send the audio stream directly to the Chromecast (i.e. your "Web Receiver" app) from the AM servers, rather than streaming the file via Cider as an intermediary please? i.e. Cider being just the controller, and the your Chromecast "web receiver" app communicating with the servers directly. Other services seem to do this though, but not sure if it's possible with MusicKit. Thanks.

**Comment by quacksire** [comment]
This is not possible with the MusicKit JS  API

**Comment by grolschie** [comment]
Ok. Thank you anyway. :-)
